transmart-rest-api/src/main/groovy/org/transmartproject/rest/serialization/tabular/TabularResultSPSSSerializer.groovy

```diff
@@ -224,13 +224,20 @@ class TabularResultSPSSSerializer implements TabularResultSerializer {
         outputStream << buffer
     }
 
-    private static String missingValueExpression(MissingValues missingValues) {
+    private static String missingValueExpression(VariableDataType type, MissingValues missingValues) {
         List<String> parts = []
         if (missingValues.lower || missingValues.upper) {
             parts.add((missingValues.lower as String ?: 'LOWEST') + ' THRU ' + (missingValues.upper as String ?: 'HIGHEST'))
         }
         if (missingValues.values) {
-            parts.add(missingValues.values.join(', '))
+            def valuesText = missingValues.values.collect({ value ->
+                if (type == VariableDataType.STRING) {
+                    quote(value as String)
```

Comment:
so missing values can be of type string, numeric or date? Dates don't have to be in quotes?

Reply:
For String I encountered this specific issue, for dates I am not sure.